packages/fuels/tests/harness.rs

```diff
@@ -3805,3 +3827,141 @@ async fn test_script_interface() -> Result<(), Error> {
 
     Ok(())
 }
+
+#[tokio::test]
+async fn test_vector() -> Result<(), Error> {
+    abigen!(
+        MyContract,
+        "packages/fuels/tests/test_projects/vectors/out/debug/vectors-abi.json"
+    );
+
+    let wallet = launch_provider_and_get_wallet().await;
+
+    let contract_id = Contract::deploy(
+        "tests/test_projects/vectors/out/debug/vectors.bin",
+        &wallet,
+        TxParameters::default(),
+        StorageConfiguration::default(),
+    )
+    .await?;
+
+    let contract_instance = MyContractBuilder::new(contract_id.to_string(), wallet).build();
```

Comment:
You will have to change if because of the `methods()` PR. might a s well use the `setup_contract_test!` macro.